src/compounds/Document/Document.stories.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import Document from './Document'
+
+export default {
+  title: 'Components/Document',
+  component: Document,
+}
+
+const Template = (args) => <Document {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  id: '76EBF1',
+  date: 'Nov 22 2022',
+  documentStatus: 'Approval Required',
+  documentStatusColor: 'bg-secondary',
+  documentType: 'SOW',
+  documentTypeColor: 'bg-dark',
+  lineItems: [
+    {
+        "id": 921424,
+        "retail_unit_price_currency": "$1.00",
+        "retail_unit_price": "1.0",
+        "retail_subtotal_price_currency": "$1.00",
+        "retail_subtotal_price": "1.0",
+        "quantity": "1.0",
+        "currency": "USD",
+        "currency_unit": "$",
+        "name": "Milestone 1",
+        "subtotal": "$1.00",
+        "unit_price": "$1.00"
+    },
+    {
+        "id": 921425,
+        "retail_unit_price_currency": "$2.00",
+        "retail_unit_price": "2.0",
+        "retail_subtotal_price_currency": "$2.00",
+        "retail_subtotal_price": "2.0",
+        "quantity": "1.0",
+        "currency": "USD",
+        "currency_unit": "$",
+        "name": "Milestone 2",
+        "subtotal": "$2.00",
+        "unit_price": "$2.00"
+    }
+  ],
```

Comment:
can you create this as a variable in args.js and import it back here and in the line items table instead?